### PR TITLE
xzdiff: add page

### PR DESCRIPTION
**Pull request by tonytheleg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
xzdiff (XZ Utils) 5.2.5

For #8859

### CODE REVIEW

File: pages/common/xzdiff.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# xzdiff
+
+> Invokes the diff command on files compressed with xz, lzma, gzip, bzip2, lzop, or zstd.
+> All options specified are passed directly to diff.
+> More information: <https://manned.org/xzdiff.1>.
```
Comment (navarroaxel) [inline pages/common/xzdiff.md:5]:
```suggestion
> More information: <https://manned.org/xzdiff>.
```

File: pages/common/xzdiff.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`xzdiff {{path/to/file1}} {{path/to/file2}}`
+
+- Compare files, showing the differences side by side:
+
+`xzdiff -y {{path/to/file1}} {{path/to/file2}}`
+
+- Compare files and report only that they differ (no details on what is different):
+
+`xzdiff -q {{path/to/file1}} {{path/to/file2}}`
```
Comment (navarroaxel) [inline pages/common/xzdiff.md:17]: We prefer large arguments if available
```suggestion
`xzdiff --brief {{path/to/file1}} {{path/to/file2}}`
```

File: pages/common/xzdiff.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+> All options specified are passed directly to diff.
+> More information: <https://manned.org/xzdiff.1>.
+
+- Compare files:
+
+`xzdiff {{path/to/file1}} {{path/to/file2}}`
+
+- Compare files, showing the differences side by side:
+
+`xzdiff -y {{path/to/file1}} {{path/to/file2}}`
```
Comment (navarroaxel) [inline pages/common/xzdiff.md:13]:
```suggestion
`xzdiff --side-by-side {{path/to/file1}} {{path/to/file2}}`
```

File: pages/common/xzdiff.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Compare files and report only that they differ (no details on what is different):
+
+`xzdiff -q {{path/to/file1}} {{path/to/file2}}`
+
+- Compare files and report when the files are the same:
+
+`xzdiff -s {{path/to/file1}} {{path/to/file2}}`
```
Comment (navarroaxel) [inline pages/common/xzdiff.md:21]:
```suggestion
`xzdiff --report-identical-files {{path/to/file1}} {{path/to/file2}}`
```

File: pages/common/xzdiff.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- Compare files and paginate results:
+
+`xzdiff -l {{path/to/file1}} {{path/to/file2}}`
+
+- Compare directories recursively (shows names for differing files/directories as well as changes made to files):
+
+`diff -r {{path/to/file1}} {{path/to/file2}}`
```
Comment (navarroaxel) [inline pages/common/xzdiff.md:29]:
```suggestion
`diff --recursive {{path/to/file1}} {{path/to/file2}}`
```

File: pages/common/xzdiff.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+`xzdiff -q {{path/to/file1}} {{path/to/file2}}`
+
+- Compare files and report when the files are the same:
+
+`xzdiff -s {{path/to/file1}} {{path/to/file2}}`
+
+- Compare files and paginate results:
+
+`xzdiff -l {{path/to/file1}} {{path/to/file2}}`
```
Comment (navarroaxel) [inline pages/common/xzdiff.md:25]:
```suggestion
`xzdiff --paginate {{path/to/file1}} {{path/to/file2}}`
```

File: pages/common/xzdiff.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+- Compare files and report only that they differ (no details on what is different):
+
+`xzdiff -q {{path/to/file1}} {{path/to/file2}}`
+
+- Compare files and report when the files are the same:
+
+`xzdiff -s {{path/to/file1}} {{path/to/file2}}`
+
+- Compare files and paginate results:
```
Comment (navarroaxel) [inline pages/common/xzdiff.md:23]:
```suggestion
- Compare files using paginated results:
```

**Review by einverne — APPROVED** [review]
Looks good to me after @navarroaxel 's suggestion.

**Comment by tonytheleg** [comment]
@einverne all requested changes made. Thanks!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**